lib/src/universal_ble.dart

```diff
@@ -225,6 +225,13 @@ class UniversalBle {
     );
   }
 
+  /// On Web, it returns true if the web browser supports [watchAdvertisements]. The rest of the platforms will always return true.
+  /// If [canWatchAdvertisements] returns true on `Web`, then you will get scanResult updates of the selected device.
+  /// Not every browser supports this API yet. It is hidden behind the `chrome://flags/#enable-experimental-web-platform-features` flag.
+  /// Even if the device technically has the method, sometimes it won't update ScanResults even though the device may be sending them.
```

Comment:
This line is unclear

Reply:
This is basically for Web, to simply tell us if the Browser can update ScanResults of the selected device ( Device we select from that Web Dialog ), then if the browser supports `watchAdvertisement` then we can wait for the required updates like Manufacturer data or services to be updated by the browser
Note: Most of the browsers need experimental flag for this feature